examples/simulation-tensorflow-tabular/sim.py

```diff
@@ -0,0 +1,249 @@
+import os
+import argparse
+from typing import Dict, List, Tuple
+
+import tensorflow as tf
+
+import flwr as fl
+from flwr.common import Metrics
+from flwr.simulation.ray_transport.utils import enable_tf_gpu_growth
+
+from datasets import Dataset
+from flwr_datasets import FederatedDataset
+from flwr_datasets.partitioner import DirichletPartitioner
+
+import pandas as pd
+
+from datasets import load_dataset
+from sklearn.preprocessing import StandardScaler
+from sklearn.model_selection import train_test_split
+
+# Make TensorFlow logs less verbose
+os.environ["TF_CPP_MIN_LOG_LEVEL"] = "3"
+
+parser = argparse.ArgumentParser(description="Flower Simulation with Tensorflow/Keras for tabular dataset")
+
+parser.add_argument(
+    "--num_cpus",
+    type=int,
+    default=1,
+    help="Number of CPUs to assign to a virtual client",
+)
+parser.add_argument(
+    "--num_gpus",
+    type=float,
+    default=0.0,
+    help="Ratio of GPU memory to assign to a virtual client",
+)
+
+VERBOSE = 0
+NUM_CLIENTS = 5
+RANDOM_STATE = 42
+NUM_ROUNDS = 30
+
+# Define Alpha to use for DirichletPartitioner
+ALPHA = 1000
```

Comment:
Basically, there is no point in using Dirichlet with such high value unless **heavily empathized**  reasoning. That's close to iid.

Reply:
I changed it to 0.5.

PS: Check the new [PR #3633](https://github.com/adap/flower/pull/3633) with all the modifications.